Libplanet/Blockchain/Policies/BlockPolicy.cs

```diff
@@ -29,28 +37,32 @@ public class BlockPolicy<T> : IBlockPolicy<T>
         /// <param name="difficultyBoundDivisor">Configures
         /// <see cref="DifficultyBoundDivisor"/>. 128 by default.</param>
         public BlockPolicy(
+            IEnumerable<TBlockAction> blockActions = null,
             int blockIntervalMilliseconds = 5000,
             long minimumDifficulty = 1024,
             int difficultyBoundDivisor = 128)
             : this(
+                blockActions,
                 TimeSpan.FromMilliseconds(blockIntervalMilliseconds),
                 minimumDifficulty,
                 difficultyBoundDivisor)
         {
         }
 
         /// <summary>
-        /// Creates a <see cref="BlockPolicy{T}"/> with configuring
+        /// Creates a <see cref="BlockPolicy{TxAction, TBlockActionT}"/> with configuring
         /// <see cref="BlockInterval"/>, <see cref="MinimumDifficulty"/> and
         /// <see cref="DifficultyBoundDivisor"/>.
         /// </summary>
+        /// <param name="blockActions">A list of <see cref="IAction"/>s for a miner.</param>
         /// <param name="blockInterval">Configures <see cref="BlockInterval"/>.
         /// </param>
         /// <param name="minimumDifficulty">Configures
         /// <see cref="MinimumDifficulty"/>.</param>
         /// <param name="difficultyBoundDivisor">Configures
         /// <see cref="DifficultyBoundDivisor"/>.</param>
         public BlockPolicy(
+            IEnumerable<TBlockAction> blockActions,
```

Comment:
How about limiting a single block action for `BlockPolicy`?

Reply:
Do you mean `blockAction` instead of `blockActions`? It seems like that would be good for the current use case. Opinion? @dahlia

Reply:
> Do you mean blockAction instead of blockActions?

yes, I did.

Reply:
I changed `BlockActions` list to `BlockAction` which is a single action.